tests/test_json.py

```diff
@@ -9,12 +9,8 @@
 
 """
 import unittest
-try:
-    import ujson
-    from ujson import JSONDecodeError
-except ImportError:
-    import json as ujson
-    from json import JSONDecodeError
+import ujson
+from json import JSONDecodeError
```

Comment:
Is there no difference between JSONDcodeError from ujson and json? Even so, I suggest to use 
```suggestion
from ujson import JSONDecodeError
```
to be in line with everything